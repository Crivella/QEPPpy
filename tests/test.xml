<?xml version = "1.0" encoding = "UTF-8" ?>
<root version="1.0">
	<t1 a1="1">
		<t3 b1="1" b2="2" b3="3">
		</t3>
	</t1>
	<t2 a1="2">
		<t3 b1="1" b2="2" b3="3">
		</t3>
	</t2>
	<t5>
		<t6>1</t6>
		<t6>2</t6>
		<t6>3</t6>
		<t6>4</t6>
		<t6>5</t6>
		<t6>6</t6>
		<t6>7</t6>
		<t6>8</t6>
		<t6>9</t6>
	</t5>
	<t7>
		<t8 c1="1">1</t8>
		<t8 c1="1">2</t8>
		<t8 c1="1">3</t8>
		<t8 c1="1">4</t8>
		<t8 c1="1">5</t8>
		<t8 c1="1">6</t8>
		<t8 c1="1">7</t8>
		<t8 c1="1">8</t8>
		<t8 c1="1">9</t8>
	</t7>
	<t9>
		<t10>1</t10>
		<t10>2</t10>
		<t10>3</t10>
	</t9>
	<t9>
		<t10>1</t10>
		<t10>2</t10>
		<t10>3</t10>
	</t9>	
</root>