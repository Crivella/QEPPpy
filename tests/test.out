val_int =  125
val_float   = 2.3
val_str  = non saprei
multiline_1 = 1, 2, 3
multiline_2 = 1, 2, 3
multiline_3 = 1, 2, 3
multiline_4 = 1, 2, 3
multiline_5 = 1, 2, 3
multiline_6 = 1, 2, 3